### ISSUE TITLE
Overridependingtransition not working on Android 12

### ISSUE THREAD

**Issue by Massi-X** [AppIntro/AppIntro#985]
## 🐛 Describe the bug
Hi, this is a minor bug affecting Android 12 only. The AppIntro activity is started by an intent with the usual `startActivity(context, class)`, immediately followed by `overridePendingTransition(0, 0)` (so no animation). It works fine for every Android version but 12. Can't find where the problem resides. Obviously with other activities (that extends AppCompatActivity for example) everything works fine.

## ⚠️ Current behavior
The default activity switch animation is played (slide to the left).

## ✅ Expected behavior
The overridden transition should be played.

## 💣 Steps to reproduce
Make a basic activity and a basic AppIntro activity, can both be empty. Make the main activity launch the AppIntro activity with the code below:
```
startActivity(this, AppIntro.class);
overridePendingTransition(0, 0);
```
Test this on an Android 12 device. The slide animation is played instead of no animation.

## 📷 Screenshots
Not applicable

## 📑 Your Code
See above.

## 📱 Tech info
 - AppIntro Version: 6.1.0
 - Device: Emulator
 - Android OS Version: 12 (S)

**Comment by cortinico** [comment]
I've just tested this on a Android 12 emulator and `overridePendingTransition` works correctly, 
so I'm not able to reproduce your issue.

Moreover your code here has a couple of problems, so you should probably check it:

> ```
> startActivity(this, AppIntro.class);
> overridePendingTransition(0, 0);
> ```

1. There is no `startActivity(Context, Class<?>)` method. I suppose you have an `Intent` somewhere and maybe you forgot to copy-paste it?
2. You should not start the `AppIntro` class directly (it's abstract). You should instead invoke your subclass (I suppose this is also a copy-paste mistake).

If you could provide a reproducer I could look further into this.

**Comment by Massi-X** [comment]
@cortinico yeah I wrote it down wrong, it is obviously `startActivity(intent)`.
And I'm extending Appintro2 specifically.

I did all by hearth on the phone this is why all these typos.
My code is working fine (using it for almost 4 years I think), don't know if it is a problem of my setup but I tried messing around (by replacing the intent with another one that starts an simple base activity) and the animation worked fine. Give me some time to try again and I will post here some code too

Thank you for your work anyway!
